src/System.ValueTuple/pkg/System.ValueTuple.pkgproj

```diff
@@ -3,13 +3,18 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
```

Comment:
@ericstj and I were talking about this today and we think we can completely remove this package now.